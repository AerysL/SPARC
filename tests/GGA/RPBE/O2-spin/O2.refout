***************************************************************************
*                           SPARC (May 11, 2020)                          *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                   Start time: Sun May 17 23:27:05 2020                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 21 18 18 
LATVEC:
1 0 0 
0 1 0 
0 0 1 
FD_GRID: 105 90 90
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
SPIN_TYP: 2
ELEC_TEMP_TYPE: Fermi-Dirac
SMEARING: 0.001000000336
CHEB_DEGREE: 35
NSTATES: 10
#WARNING: Number of bands may be insufficient for efficient SCF convergence.
EXCHANGE_CORRELATION: GGA_RPBE
CALC_STRESS: 1
CHEFSI_BOUND_FLAG: 0
MAXIT_SCF: 100
MINIT_SCF: 3
MAXIT_POISSON: 3000
TOL_SCF: 1.00E-06
POISSON_SOLVER: AAR
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: density
MIXING_PRECOND: kerker
TOL_PRECOND: 4.00E-05
PRECOND_KERKER_KTF: 1
PRECOND_KERKER_THRESH: 0.1
MIXING_PARAMETER: 0.3
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART: 0
REFERENCE_CUTOFF: 0.5
RHO_TRIGGER: 4
FIX_RAND: 0
VERBOSITY: 1
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: RPBE/O2-spin/O2
***************************************************************************
                           Parallelization                                 
***************************************************************************
NP_SPIN_PARAL: 2
NP_KPOINT_PARAL: 1
NP_BAND_PARAL: 10
NP_DOMAIN_PARAL: 1 1 1
NP_DOMAIN_PHI_PARAL: 4 2 3
EIG_SERIAL_MAXNS: 2000
***************************************************************************
                             Initialization                                
***************************************************************************
Number of processors               :  24
Mesh spacing                       :  0.2 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  RPBE/O2-spin/O2.out_1
Total number of atom types         :  1
Total number of atoms              :  2
Total number of electrons          :  12
Atom type 1  (valence electrons)   :  O 6
Pseudopotential                    :  RPBE/O2-spin/O.psp8
Pseudocharge radii of atom type 1  :  7.00 7.00 7.00 (x, y, z dir)
Number of atoms of type 1          :  2
Estimated total memory usage       :  1.09 GB
Estimated memory per processor     :  46.50 MB
========================================================================================
                    Self Consistent Field (SCF#1)                     
========================================================================================
Iteration     Free Energy (Ha/atom)    Magnetization     SCF Error        Timing (sec)
1            -1.5876429705E+01         2.0000E+00        1.488E-01        4.074
2            -1.6005524073E+01         2.0000E+00        3.127E-01        1.792
3            -1.5934121765E+01         2.0000E+00        1.082E-01        1.764
4            -1.5933940931E+01         2.0000E+00        8.797E-02        1.804
5            -1.5937806794E+01         2.0000E+00        1.062E-01        1.753
6            -1.5934184708E+01         2.0000E+00        3.080E-02        1.643
7            -1.5934071116E+01         2.0000E+00        2.602E-02        1.949
8            -1.5933853722E+01         2.0000E+00        4.424E-03        1.591
9            -1.5933853654E+01         2.0000E+00        4.889E-03        1.551
10           -1.5933848199E+01         2.0000E+00        1.166E-03        1.540
11           -1.5933848375E+01         2.0000E+00        5.112E-04        1.427
12           -1.5933848529E+01         2.0000E+00        3.085E-04        1.416
13           -1.5933848623E+01         2.0000E+00        1.037E-04        1.360
14           -1.5933848642E+01         2.0000E+00        5.782E-05        1.394
15           -1.5933848636E+01         2.0000E+00        5.068E-05        1.314
16           -1.5933848641E+01         2.0000E+00        2.441E-05        1.799
17           -1.5933848644E+01         2.0000E+00        1.149E-05        1.225
18           -1.5933848654E+01         2.0000E+00        2.747E-06        1.164
19           -1.5933848652E+01         2.0000E+00        1.006E-06        1.100
20           -1.5933848635E+01         2.0000E+00        5.367E-07        0.827
Total number of SCF: 20    

k = [0.000, 0.000, 0.000]
Occupation of state 9 = 0.000000000000000.
Occupation of state 10 = 0.000000000000000.
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -1.5933848636E+01 (Ha/atom)
Total free energy                  : -3.1867697271E+01 (Ha)
Band structure energy              : -6.5948632382E+00 (Ha)
Exchange correlation energy        : -6.6994894784E+00 (Ha)
Self and correction energy         : -6.4063409725E+01 (Ha)
Entropy*kb*T                       : -3.4771624327E-13 (Ha)
Fermi level                        : -2.6317409926E-01 (Ha)
RMS force                          :  1.6030182077E-01 (Ha/Bohr)
Maximum force                      :  1.6030182077E-01 (Ha/Bohr)
Time for force calculation         :  0.028 (sec)
Pressure                           : -7.0032159198E-01 (GPa)
Maximum stress                     :  2.0827277267E+00 (GPa)
Time for stress calculation        :  0.064 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  33.744 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Collaborators: J.E. Pask (LLNL), A.J. Medford (GT), E. Chow (GT)    *
*  Acknowledgements: U.S. DOE (DE-SC0019410); U.S. NSF (1333500,1553212)  *
***************************************************************************
                                                                           
