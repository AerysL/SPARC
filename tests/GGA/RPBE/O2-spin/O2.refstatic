***************************************************************************
                            Atom positions                                 
***************************************************************************
Fractional coordinates of O:
      0.4285714286       0.5000000000       0.5000000000
      0.5714285714       0.5000000000       0.5000000000
Total free energy (Ha): -3.186769727106565E+01
Atomic forces (Ha/Bohr):
  1.6030182077E-01  -8.2299621615E-08  -8.3570002630E-08
 -1.6030182077E-01   8.2299621615E-08   8.3570002630E-08
Stress (GPa): 
  2.0827277267E+00  -2.5663296723E-06  -3.5982731340E-06 
 -2.5663296723E-06   9.1185127504E-03   2.3528839563E-07 
 -3.5982731340E-06   2.3528839563E-07   9.1185364983E-03
