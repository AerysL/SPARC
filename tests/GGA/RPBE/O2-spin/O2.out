***************************************************************************
*                           SPARC (Nov 06, 2019)                          *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                 Start time: Thu Dec 19 14:53:45 2019                    *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 21 18 18 
LATVEC:
1 0 0 
0 1 0 
0 0 1 
FD_GRID: 105 90 90
FD_ORDER: 12
BOUNDARY_CONDITION: 2
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
SPIN_TYP: 2
ELEC_TEMP_TYPE: Fermi-Dirac
SMEARING: 0.001000000001
CHEB_DEGREE: 35
NSTATES: 10
EXCHANGE_CORRELATION: GGA_RPBE
CALC_STRESS: 1
CHEFSI_BOUND_FLAG: 0
MAXIT_SCF: 100
MINIT_SCF: 3
MAXIT_POISSON: 3000
TOL_SCF: 1.00E-06
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: density
MIXING_PRECOND: none
#WARNING: the preconditioner for SCF has been turned off, this 
#might lead to slow SCF convergence. To specify SCF preconditioner, 
#use 'MIXING_PRECOND' in the .inpt file
MIXING_PARAMETER: 0.3
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART: 0
REFERENCE_CUTOFF: 0.5
RHO_TRIGGER: 4
FIX_RAND: 0
VERBOSITY: 1
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: O2
***************************************************************************
                           Parallelization                                 
***************************************************************************
NP_SPIN_PARAL: 2
NP_KPOINT_PARAL: 1
NP_BAND_PARAL: 10
NP_DOMAIN_PARAL: 2 1 1
NP_DOMAIN_PHI_PARAL: 5 2 4
***************************************************************************
                             Initialization                                
***************************************************************************
Number of processors               :  40
Mesh spacing                       :  0.2 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  O2.out
Total number of atom types         :  1
Total number of atoms              :  2
Total number of electrons          :  12
Atom type 1  (valence electrons)   :  O 6
Pseudopotential                    :  ./O.psp8
Pseudocharge radii of atom type 1  :  7.20 7.20 7.20 (x, y, z dir)
Number of atoms of type 1          :  2
Fractional coordinates of atoms of type 1     :
      0.4285714286       0.5000000000       0.5000000000
      0.5714285714       0.5000000000       0.5000000000
========================================================================================
                    Self Consistent Field (SCF#1)                     
========================================================================================
Iteration     Free Energy (Ha/atom)    Magnetization     SCF Error        Timing (sec)
1            -1.5902194075E+01         2.0000E+00        1.261E-01        9.953
2            -1.5942344270E+01         2.0000E+00        1.771E-01        3.907
3            -1.5941608090E+01         2.0000E+00        1.655E-01        3.840
4            -1.5932903928E+01         2.0000E+00        5.396E-02        3.766
5            -1.5933920478E+01         2.0000E+00        1.670E-02        3.568
6            -1.5933846131E+01         2.0000E+00        6.676E-03        12.654
7            -1.5933848396E+01         2.0000E+00        1.745E-03        3.355
8            -1.5933848213E+01         2.0000E+00        7.910E-04        3.271
9            -1.5933848521E+01         2.0000E+00        3.502E-04        3.087
10           -1.5933848607E+01         2.0000E+00        1.486E-04        3.004
11           -1.5933848627E+01         2.0000E+00        5.768E-05        2.876
12           -1.5933848646E+01         2.0000E+00        2.630E-05        2.737
13           -1.5933848644E+01         2.0000E+00        1.578E-05        2.545
14           -1.5933848648E+01         2.0000E+00        5.657E-06        2.487
15           -1.5933848651E+01         2.0000E+00        6.992E-06        2.418
16           -1.5933848659E+01         2.0000E+00        2.142E-06        2.324
17           -1.5933848682E+01         2.0000E+00        1.226E-06        2.301
18           -1.5933848660E+01         2.0000E+00        8.362E-07        2.096
Total number of SCF: 18    
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -1.5933848648E+01 (Ha/atom)
Total free energy                  : -3.1867697296E+01 (Ha)
Band structure energy              : -6.5948616384E+00 (Ha)
Exchange correlation energy        : -6.6994896511E+00 (Ha)
Self and correction energy         : -6.4063409725E+01 (Ha)
Entropy*kb*T                       : -3.5940087642E-13 (Ha)
Fermi level                        : -2.6313966498E-01 (Ha)
Atomic forces (Ha/Bohr):
  1.6030113437E-01  -1.7861226545E-07  -8.4249436843E-08
 -1.6030113437E-01   1.7861226545E-07   8.4249436843E-08
RMS force                          :  1.6030113437E-01 (Ha/Bohr)
Maximum force                      :  1.6030113437E-01 (Ha/Bohr)
Time for force calculation         :  0.108 (sec)
Stress (GPa) : 
  2.0827286472E+00  -2.5568731749E-06  -1.3906495477E-06 
 -2.5568731749E-06   9.1206202942E-03  -7.0901862614E-07 
 -1.3906495477E-06  -7.0901862614E-07   9.1209750534E-03 
Pressure                           : -7.0032341420E-01 (GPa)
Maximum stress                     :  2.0827286472E+00 (GPa)
Time for stress calculation        :  0.213 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  73.043 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Collaborators: J.E. Pask (LLNL), A.J. Medford (GT), E. Chow (GT)    *
*  Acknowledgements: U.S. DOE (DE-SC0019410); U.S. NSF (1333500,1553212)  *
***************************************************************************
                                                                           
